bpf/init.sh

```diff
@@ -275,7 +275,7 @@ function bpf_compile()
 	      -DHANDLE_NS						\
 	      $EXTRA_OPTS						\
 	      -c $LIB/$IN -o - |					\
-	llc -march=bpf -mcpu=probe -mattr=dwarfris -filetype=$TYPE -o $OUT
+	llc -march=bpf -mcpu=v2 -mattr=dwarfris -filetype=$TYPE -o $OUT
```

Comment:
Problem is that this will fail verifier on older kernels where only v1 is supported. I presume that is why you saw the most recent verifier jmp opcode issue.

Reply:
right, I guess we need the same logic as master here...

Reply:
Just pulled in this,
```
 bpf, cilium: manually probe for v1/v2 BPF CPU availability

[ upstream commit 809bfff ]
```
WDYT?